core-application/src/main/java/gohigher/application/port/in/MyApplicationRequest.java

```diff
@@ -0,0 +1,18 @@
+package gohigher.application.port.in;
+
+import java.util.List;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.ToString;
+
+@Getter
+@RequiredArgsConstructor
+@ToString
+public class MyApplicationRequest {
+
+    private final String sort;
+    private final List<String> process;
+    private final List<String> scheduled;
```

Comment:
```suggestion
    private final List<Boolean> isClosed;
```

closed, unclosed 를 나타내려면 scheduled 보다 isClosed는 어떨까요? 해당 부분에 대한 Column을 isClosed로 나타낼 생각입니다!